pkg/executables/kubectl.go

```diff
@@ -2008,6 +2008,34 @@ func (k *Kubectl) ApplyTolerationsFromTaints(ctx context.Context, oldTaints []co
 	return nil
 }
 
+// PauseCAPICluster adds a `spec.Paused: true` to the CAPI cluster resource. This will cause all
+// downstream CAPI + provider controllers to skip reconciling on the paused cluster's objects
+func (k *Kubectl) PauseCAPICluster(ctx context.Context, cluster, kubeconfig string) error {
+	patch := fmt.Sprintf("{\"spec\":{\"paused\":%t}}", true)
```

Comment:
I thought the canonical way to pause a cluster was through the pause annotation?
Which also simplifies this since we won't need the merge path.

Reply:
I think the pause annotation is recommended way to pause individual objects. Most if not all controllers call the helper function over [here at CAPI](https://github.com/kubernetes-sigs/cluster-api/blob/83b8694556bc7986ee8573e93e65538957154b59/util/annotations/helpers.go#L29) in their reconcile loops to validate cluster paused. This looks for `spec.Paused` on their owner cluster or pause annotation on that specific object. If we add this spec.Pause we can get away without having to annotate all the dependents.